wavefront_lambda/README.md

```diff
@@ -0,0 +1,30 @@
+# wavefront_lambda
+
+This is a Wavefront python wrapper for AWS Lambda python function handler to send metrics directly to wavefront.
+
+## Requirements
+Python 2.7 or 3.6.
+
+## Installation
+To install from PyPi
+```
+pip install wavefront_lambda
+```
+
+## Environmental variables
+WAVEFRONT_URL = you corp wavefront url [Ex: https://yourcorp.wavefront.com]  
+WAVEFRONT_API_TOKEN = your wavefront api authentication token  
```

Comment:
For URL, let's just say: `https://<INSTANCE>.wavefront.com`.
For token: Wavefront API token with Direct Data Ingestion permission.